modules/command-engine/core/src/main/scala/surge/internal/domain/SurgeMessagePipeline.scala

```diff
@@ -109,9 +113,16 @@ private[surge] abstract class SurgeMessagePipeline[S, M, E](
       signalBus = signalBus,
       config = config)
 
-    system.actorOf(
-      KafkaClusterShardingRebalanceListener
-        .props(stateChangeActor, partitionToKafkaProducerActor, businessLogic.kafka.stateTopic.name, businessLogic.kafka.streamsApplicationId))
+    try {
+      system.actorOf(
+        KafkaClusterShardingRebalanceListener
+          .props(stateChangeActor, partitionToKafkaProducerActor, businessLogic.kafka.stateTopic.name, businessLogic.kafka.streamsApplicationId),
+        "kafka-cluster-rebalance-listener")
```

Comment:
Yeah, it is only created when we have enabled the Akka cluster approach. 
And Akka Cluster doesn't work with multiple engines, because we use a single actor system which leads to issues with the co-location of shards with different Kafka topics (partitions).